corehq/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -18,31 +18,56 @@ hqDefine("dashboard/js/dashboard", [
         self.helpText = options.help_text;
         self.hasError = ko.observable(false);
 
-        self.hasItemList = options.pagination && options.pagination.pages;
-        if (self.hasItemList) {
-            self.itemsPerPage = options.pagination.items_per_page;
-            self.totalPages = options.pagination.pages;
+        // Might get updated if this tile supports an item list but it's empty
+        self.hasItemList = ko.observable(options.has_item_list);
+
+        if (self.hasItemList()) {
+            self.itemsPerPage = 5;
             self.currentPage = ko.observable();         // 1-indexed
             self.pageList = ko.observableArray();
+
+            // Set via ajax
+            self.totalPages = ko.observable();
             self.items = ko.observableArray();
         }
 
         // Control visibility of various parts of tile content
         self.showBackgroundIcon = ko.computed(function() {
-            return self.hasItemList && !self.hasError();
+            return self.hasItemList() && !self.hasError();
         });
         self.showSpinner = ko.computed(function() {
-            return self.hasItemList && self.items().length === 0 && !self.hasError();
+            return self.hasItemList()                                   // this is an ajax tile
+                   && (self.items().length === 0 || !self.totalPages()) // both requests are complete...
```

Comment:
does `!self.totalPages()` need to distinguish between `undefined` and `[]`?

Reply:
No. This is really just checking whether or not `totalPages` is undefined (versus being an integer). Updated code & comment to clarify.